src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        class="card__image"
+        alt="APPLE A1419 iMac 27"
+      >
+      <div class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="card__product-code">Product code: 195434</div>
+      <div class="stars">
+        <div class="stars__elem stars__elem--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="stars__reviews">Reviews: 5</div>
+      </div>
```

Comment:
following BEM
```suggestion
      <div class="card__rating">
        <div class="card__stars stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <div class="card__reviews">Reviews: 5</div>
      </div>
```

Reply:
That is, the views item is not included in the starred block? 

Why do we need to use item "card__stars", if we can use only block "stars" and modifier "stars--_**n**_"?

Reply:
changed, but at my discretion